src/lib/openapi/spec/project-feature-schema.ts

```diff
@@ -0,0 +1,99 @@
+import type { FromSchema } from 'json-schema-to-ts';
+import { tagSchema } from './tag-schema';
+import { projectFeatureEnvironmentSchema } from './project-feature-environment-schema';
+
+export const projectFeatureSchema = {
+    $id: '#/components/schemas/projectFeatureSchema',
+    type: 'object',
+    additionalProperties: false,
+    required: [
+        'name',
+        'type',
+        'description',
+        'stale',
+        'favorite',
+        'impressionData',
+        'createdAt',
+        'lastSeenAt',
+        'environments',
+    ],
+    description: 'A project feature flag definition',
+    properties: {
+        name: {
+            type: 'string',
+            example: 'disable-comments',
+            description: 'Unique feature name',
+        },
+        type: {
+            type: 'string',
+            example: 'kill-switch',
+            description:
+                'Type of the flag e.g. experiment, kill-switch, release, operational, permission',
+        },
+        description: {
+            type: 'string',
+            nullable: true,
+            example:
+                'Controls disabling of the comments section in case of an incident',
+            description: 'Detailed description of the feature',
+        },
+        stale: {
+            type: 'boolean',
+            example: false,
+            description:
+                '`true` if the feature is stale based on the age and feature type, otherwise `false`.',
+        },
+        favorite: {
+            type: 'boolean',
+            example: true,
+            description:
+                '`true` if the feature was favorited, otherwise `false`.',
+        },
+        impressionData: {
+            type: 'boolean',
+            example: false,
+            description:
+                '`true` if the impression data collection is enabled for the feature, otherwise `false`.',
+        },
+        createdAt: {
+            type: 'string',
+            format: 'date-time',
+            nullable: true,
+            example: '2023-01-28T15:21:39.975Z',
+            description: 'The date the feature was created',
+        },
+        lastSeenAt: {
+            type: 'string',
+            format: 'date-time',
+            nullable: true,
+            deprecated: true,
+            example: '2023-01-28T16:21:39.975Z',
+            description:
+                'The date when metrics where last collected for the feature. This field was deprecated in v5, use the one in featureEnvironmentSchema',
```

Comment:
What is `featureEnvironmentSchema`? Is it projectFeatureEnvironmentSchema?

Isn't it better to point the user to where they'll find what they're looking for. How's this?

```suggestion
                'The date and time when metrics where last collected for this flag in any environment. This field was deprecated in v5. You should instead use the `lastSeenAt` property on the individual environments listed under the `environments` property.',
```

Reply:
Your suggestion is spot on. Applying it now